cmd/opampsupervisor/e2e_test.go

```diff
@@ -273,6 +280,42 @@ func TestSupervisorStartsCollectorWithRemoteConfig(t *testing.T) {
 	}, 10*time.Second, 500*time.Millisecond, "Log never appeared in output")
 }
 
+func TestSupervisorStartsCollectorWithLocalConfigOnly(t *testing.T) {
```

Comment:
If this is related to starting the Supervisor with only a local config and not with a remote config, could we also check the pipeline stared by the Collector by reading the files returned by `createSimplePipelineCollectorConf`?

Reply:
Sure, will fix this.

Reply:
Fixed in [`54ee558` (#39949)](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39949/commits/54ee558c319bc502edf9793f737bb455fd80bb38)

Reply:
Looks good, thanks.